ansible/install-apps.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+- name: Install OS-level apps
+  hosts: all
+  become: yes
+  vars:
+  - apt_file: /etc/apt/sources.list.d/google-chrome.list
+  - apt_file_Git: 
+  tasks:
+  - name: Does the Google apt file exist?
+    command: test -f {{apt_file}}
```

Comment:
But in general, best practice is to use existing ansible modules. Because they do internal idempotency checks and give you structured result back instead of some unparsed stdout/stderr strings that you'd have to analyze manually.

Reply:
Here's a list of fs-related modules: https://docs.ansible.com/ansible/latest/modules/list_of_files_modules.html.

I think you should use https://docs.ansible.com/ansible/latest/modules/stat_module.html#return-values.
It'll give you a nested result which you can then reuse.

Also, there's a high chance the module which you'd use to create this file already does such idempotency check internally. (At least it should in general case)